README.md

```diff
@@ -0,0 +1,172 @@
+# Browser Error Collection Notifications
+
+This is a prototype service that reads processed events from Sentry (exported via Amazon SQS) and evaluates several rules to determine if it should send email alerts.
+
+## Development Setup
+
+Prerequisites:
+
+- Docker 18.03.0
+- docker-compose 1.21.0
+
+1. Clone the repository:
+
+   ```sh
+   git clone https://github.com/mozilla/bec-alerts.git
+   cd bec-alerts
+   ```
+2. Build the Docker image:
+
+   ```sh
+   docker-compose build
+   ```
+3. Initialize Sentry and create an admin account (requires user input):
+
+   ```sh
+   docker-compose run sentry sentry upgrade
+   ```
+4. Initialize processor database:
+
+   ```sh
+   docker-compose run processor bec-alerts manage migrate
```

Comment:
Before this step, you need to start the `processor-postgres` service and wait for it to come up fully. Otherwise you get:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/base/base.py", line 216, in ensure_connection
    self.connect()
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/base/base.py", line 194, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/postgresql/base.py", line 168, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/usr/local/lib/python3.6/site-packages/psycopg2/__init__.py", line 130, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not connect to server: Connection refused
	Is the server running on host "processor-postgres" (172.23.0.5) and accepting
	TCP/IP connections on port 5432?
```

until it's up.